src/fides/api/ops/email_templates/templates/consent_request_email_fulfillment.html

```diff
@@ -34,7 +34,12 @@
                      {%- endfor %}
                      <td>
                         {% for consent_option in requested_change.consent_preferences -%}
-                           {{consent_option.data_use}}: {{"Opt-in" if consent_option.opt_in else "Opt-out"}}{{", " if loop.index < requested_change.consent_preferences|length else ""}}
+                           {% if consent_option.data_use %}
+                              {{consent_option.data_use}}: {{"Opt-in" if consent_option.opt_in else "Opt-out"}}{{", " if loop.index < requested_change.consent_preferences|length else ""}}
+                           {% endif %}
+                           {% if consent_option.privacy_notice_history %}
+                              {{consent_option.privacy_notice_history.name}}: {{"Opt-in" if consent_option.opt_in else "Opt-out"}}{{", " if loop.index < requested_change.consent_preferences|length else ""}}
+                           {% endif %}
```

Comment:
Email template accommodates both consent being saved w.r.t data use (old) and privacy notice (new).

I'm putting the "name" of the privacy notice in the email, (actually of the history record) and enforcing that the name exists.  This should be fine if customers use our names - looks like they'll be things like "Advertising". If the customer creates their own custom ones, it's possible we send confusing text in this email.

Currently this email template is just used for the sovrn connector.